botwfstools/botw_patcher.py

```diff
@@ -15,10 +15,19 @@
 import sys
 import typing
 import wszst_yaz0
+import re
 
 ARCHIVE_EXTS = {'sarc', 'pack', 'bactorpack', 'bmodelsh', 'beventpack', 'stera', 'stats',
                 'ssarc', 'spack', 'sbactorpack', 'sbmodelsh', 'sbeventpack', 'sstera', 'sstats',
                 'blarc', 'sblarc', 'genvb', 'sgenvb'}
+AOC_PREFIX = 'Aoc/0010/'
+AOC_PREFIX_CONDITIONS = {'Terrain\/A\/AocField.*', 'Pack\/.*.pack', 'UI\/StaffRollDLC\/.*',
+                'Map\/MainField\/.*', 'Map\/MainFieldDungeon\/.*', 'Map\/AocField\/.*', 'Map\/CDungeon\/.*',
+                'Physics\/StaticCompound\/AocField\/.*', 'Physics\/StaticCompound\/MainFieldDungeon\/.*',
+                'Physics\/StaticCompound\/CDungeon\/.*', 'Movie\/Demo/.*', 'Game\/AocField\/.*', 'NavMesh\/AocField\/.*',
+                'NavMesh\/MainFieldDungeon\/.*', 'NavMesh\/CDungeon\/.*', 'Physics\/TeraMeshRigidBody\/AocField\/.*',
+                'Voice\/.*\/Stream_Demo\/.*\/.*\.bfstm', 'System\/AocVersion\.txt'}
+RE_PATTERNS = {re.compile(string) for string in AOC_PREFIX_CONDITIONS}
```

Comment:
This doesn't have to be a set either. A set is a specific data structure you use when you need fast inserts, removals or "is element in the set" checks. You're just iterating over the compiled regexp objects so a simple list will do.